src/collections/vec.rs

```diff
@@ -1983,26 +1983,23 @@ __impl_slice_eq1! { Vec<'a, A>, &'b [B] }
 __impl_slice_eq1! { Vec<'a, A>, &'b mut [B] }
 // __impl_slice_eq1! { Cow<'a, [A]>, Vec<'b, B>, Clone }
 
-macro_rules! array_impls {
-    ($($N: expr)+) => {
-        $(
-            // NOTE: some less important impls are omitted to reduce code bloat
-            __impl_slice_eq1! { Vec<'a, A>, [B; $N] }
-            __impl_slice_eq1! { Vec<'a, A>, &'b [B; $N] }
-            // __impl_slice_eq1! { Vec<A>, &'b mut [B; $N] }
-            // __impl_slice_eq1! { Cow<'a, [A]>, [B; $N], Clone }
-            // __impl_slice_eq1! { Cow<'a, [A]>, &'b [B; $N], Clone }
-            // __impl_slice_eq1! { Cow<'a, [A]>, &'b mut [B; $N], Clone }
```

Comment:
I removed this, since it wouldn't even compile (both `Cow` and arrays are defined in a different module), I'm not sure why it was there in the first place.